imf/imf.py

```diff
@@ -172,8 +172,8 @@ def m_integrate(self, mlow, mhigh, numerical=False, **kwargs):
         """
         Integrate the mass function over some range
         """
-        if mhigh <= mlow:
-            raise ValueError("Must have mlow < mhigh in integral")
+        if mhigh < mlow:
+            raise ValueError("Must have mlow <= mhigh in integral")
```

Comment:
what's the logic here?  I don't think the function should work if `mlow == mhigh` - that's just a delta function and not well-handled, right?

Reply:
essentially it's a bracketing problem with the root finding; if the bracket includes one of the mass limits, this throws up an error there. I can shift the brackets in slightly, but I thought it would be better not to do that where it's not strictly necessary. integrating a function over a range of size zero should just be zero, right?